var/spack/repos/builtin/packages/fribidi/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Fribidi(AutotoolsPackage):
+    """GNU FriBidi: The Free Implementation of the Unicode Bidirectional
+    Algorithm."""
+
+    homepage = "https://github.com/fribidi/fribidi"
+    url      = "https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2"
+
+    version('1.0.5', sha256='6a64f2a687f5c4f203a46fa659f43dd43d1f8b845df8d723107e8a7e6158e4ce')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    def url_for_version(self, version):
+        url  = 'https://github.com/fribidi/fribidi/releases/download/'
+        url += 'v{0}/fribidi-{0}.tar.bz2'
+        return url.format(version)
```

Comment:
This package doesn't actually need a `url_for_version`, Spack is smart enough to figure it out:
```console
$ spack url parse https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2
==> Parsing URL: https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2

==> Matched version regex  0: r'^[a-zA-Z+._-]+[._-]v?(\\d[\\d._-]*)$'
==> Matched  name   regex  0: r'github\\.com/[^/]+/([^/]+)'

==> Detected:
    https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2
                               -------                                  ~~~~~
    name:    fribidi
    version: 1.0.5

==> Substituting version 9.9.9b:
    https://github.com/fribidi/fribidi/releases/download/v9.9.9b/fribidi-9.9.9b.tar.bz2
                               -------                                   ~~~~~~
```

Reply:
Removed `url_for_version` in https://github.com/spack/spack/pull/12386/commits/91e7ea41fb28d55af3b20060a0c12bc482107879